installation.md

```diff
@@ -17,10 +17,18 @@ After installing, `clangd --version` should print `clangd version 7.0.0` or late
 
 <details>
 <summary markdown="span">Mac OS X</summary>
+ 
```

Comment:
Any reason for this empty line?

Reply:
I think (though I could be mistaken) that raw HTML in Markdown is supposed to have a blank line after it. At the very least, it makes the [file preview in GitHub look better](https://github.com/llvm/clangd-www/pull/32/files?short_path=a0ab6b4#diff-a0ab6b4c22ea1ad409f8e2271ae1c1220e49c6a2782afa92ff2462367cbb37f5).